coordinator/src/app_desc.rs

```diff
@@ -410,27 +410,37 @@ mod tests {
     fn load_essentials() {
         let source = unindent(
             r#"
-            modules:
-                awesome-module:
-                    hash: 1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
-                    transactions:
-                        - has-seq
-                    init-config:
-                        test: 1
-                        test1:
-                            key1: 1
-                            key2: sdfsdaf
-            host:
-                imports:
-                    a: awesome-module/a.a
-                    \namespace:
-                        b.b: asdfsdaf-asdf
-            transactions:
-                great-tx: awesome-module
-            param-defaults:
-                num-threads: 10
-        "#,
+[modules.awesome-module]
+hash = "1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef"
+transactions = ["has-seq"]
+
+[modules.awesome-module.init-config]
+test = 1
+
+[modules.awesome-module.init-config.test1]
+key1 = 1
+key2 = "sdfsdaf"
+
+[modules.awesome-module.exports]
+init-genesis.init-genesis = {}
+init-chain.init-chain = {}
+update-chain.update-chain = {}
+
+[host]
+
+[host.imports]
+a = "awesome-module/a.a"
+
+[host.imports."\\namespace"]
+"b.b" = "asdfsdaf-asdf"
+
+[transactions]
+great-tx = "awesome-module"
+
+[param-defaults]
+num-threads = "10"
```

Comment:
This should better be an integer? Actually no code uses the actual value of 10, BTW.

Reply:
@dynaxis The type of `param_defaults` is `Namespzed<String>`. It requires the config to be a `String` value.
I think in YAML, `num-threads: 10` can be parsed both number or string.

https://github.com/CodeChain-io/foundry/blob/fe8940a27cb5505f01b7d6bdf30b39e9b26d25b6/coordinator/src/app_desc.rs#L151

Reply:
@majecty Maybe it is better typed as `Namespaced<Value>`. Anyway, let's proceed as it is, and deal with any possible problem when something gets in our way.